test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -2534,6 +2534,55 @@ BOOST_AUTO_TEST_CASE(dev_struct_getter_override)
 	checkNatspec(sourceCode, "Thing", natspec2, false);
 }
 
+BOOST_AUTO_TEST_CASE(dev_struct_getter_override_no_return_name)
+{
+	char const *sourceCode = R"(
+		interface IThing {
+			///@return
+			function value(uint) external returns (uint128,uint128);
+		}
+
+		contract Thing is IThing {
+			struct Value {
+				uint128 x;
+				uint128 A;
+			}
+			mapping(uint=>Value) public override value;
+		}
+	)";
+
+	char const *natspec = R"ABCDEF({
+		"methods":
+		{
+			"value(uint256)":
+			{
+				"returns":
+				{
+					"_0": ""
+				}
+			}
+		}
+	})ABCDEF";
+
+	char const *natspec2 = R"ABCDEF({
+		"methods": {},
+		"stateVariables":
+		{
+			"value":
+			{
+				"return": "x ",
```

Comment:
``"x "``? Where's that coming from? Seems a bit weird.

Reply:
From the value struct. It assumed it's only one parameter and the names were different so it adjusted it

Reply:
And what about ``A``?
I'm wondering whether everything is sane here or whether we should check if this is really what we want, resp. whether there's some issue hidden here.

Reply:
Basically, the amount of (documented) parameters is inherited and the base documentation only documented one (I mean, not really, but an empty return is understood as the first parameter with an empty descrption)

Reply:
I wouldn't dive too deep into this because I think the whole structure of natspec needs a more .. structured rewrite anyway

Reply:
I'm not entirely convinced :-). Where is the space coming from then :-)? Is there a similar case that already does something equally weird on ``develop`` :-)?

Reply:
I mean, this PR is only the test, so we can merge it of course, but we should more properly look into this further before the next release, so I tend to keep it open as a reminder...

Reply:
Usually it would be `"return": "x description of parameter",` but as the description is missing, it is just a space. That's all there is to it